test/unit/views/MovementsViewSpec.scala

```diff
@@ -20,89 +20,133 @@ import java.time.format.DateTimeFormatter
 import java.time.temporal.ChronoUnit.MINUTES
 import java.time.{Instant, LocalDate, ZoneId, ZoneOffset}
 
-import base.BaseSpec
+import base.Injector
 import controllers.routes
 import models.UcrBlock
+import models.cache.ArrivalAnswers
 import models.notifications.{Entry, Notification, ResponseType}
 import models.submissions.{ActionType, Submission}
-import play.api.test.FakeRequest
 import play.twirl.api.Html
-import testdata.CommonTestData.conversationId
-import testdata.ConsolidationTestData
+import testdata.CommonTestData._
 import testdata.ConsolidationTestData._
 import testdata.MovementsTestData.exampleSubmission
 import testdata.NotificationTestData.exampleNotificationFrontendModel
-import utils.Stubs
 import views.html.movements
-import views.spec.ViewValidator
 
-class MovementsViewSpec extends BaseSpec with Stubs with ViewValidator with MessagesStub {
+class MovementsViewSpec extends ViewSpec with Injector {
 
-  private implicit val implicitFakeRequest = FakeRequest()
+  private implicit val implicitFakeRequest = journeyRequest(ArrivalAnswers())
 
   private val dateTimeFormatter = DateTimeFormatter.ofPattern("dd MMM yyyy 'at' HH:mm").withZone(ZoneId.of("Europe/London"))
   private val dateTime: Instant = LocalDate.of(2019, 10, 31).atStartOfDay().toInstant(ZoneOffset.UTC)
 
-  private def createView(submissions: Seq[(Submission, Seq[Notification])] = Seq.empty): Html =
-    new movements(mainTemplate, dateTimeFormatter)(submissions)(FakeRequest(), messages)
+  private val page = instanceOf[movements]
+
+  private def createView(submissions: Seq[(Submission, Seq[Notification])] = Seq.empty): Html = page(submissions)
 
   "Movements page" should {
+    val emptyPage = createView()
 
     "contain title" in {
+      emptyPage.getTitle must containMessage("submissions.title")
+    }
+
+    "contain back button" in {
+      emptyPage.getBackButton mustBe defined
+      emptyPage.getBackButton.get must containMessage("site.back.toStartPage")
+      emptyPage.getBackButton.get must haveHref(controllers.routes.ChoiceController.displayChoiceForm())
```

Comment:
Instead of checking if the option is defined and after that get, you can use `value` from `OptionValues` to extract data. The error if the option is empty should be legible enough.

non blocking comment